packages/neon/neon/lib/src/utils/user_agent.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:meta/meta.dart';
+
+late String? _userAgent;
+
+@internal
+set neonUserAgent(final String value) => _userAgent = value;
```

Comment:
What do you think about making this a method like:

```dart
void buildNeonUseragent(PackageInfo packageInfo){
  var buildNumber = packageInfo.buildNumber;
  if (buildNumber.isEmpty) {
    buildNumber = '1';
  }
  _userAgent = 'Neon ${packageInfo.version}+$buildNumber';
}
```

or maybe using a stringbuffer. This would cleanup neon.dart